From 2dd64f8110c09829d6f6d433058406917a8062bd Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Mon, 20 Oct 2025 11:14:11 +0100
Subject: [PATCH] Update tokio-tungstenite from 0.26 to 0.28
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This matches the verison used by the latest axum, 0.8.6.

This was proposed upstream as part of a larger set of updates, “chore:
update dependencies” at https://github.com/asciinema/asciinema/pull/698.
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 6aaba8f..6b4afc0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,7 +35,7 @@ tracing = { version = "0.1", default-features = false }
 tracing-subscriber = { version = "0.3.20", default-features = false, features = ["fmt", "env-filter"] }
 rgb = { version = "0.8", default-features = false }
 url = "2.5"
-tokio-tungstenite = { version = "0.26", default-features = false, features = ["connect", "rustls-tls-native-roots"] }
+tokio-tungstenite = { version = "0.28", default-features = false, features = ["connect", "rustls-tls-native-roots"] }
 rustls = { version = "0.23", default-features = false, features = ["ring"] }
 tokio-util = { version = "0.7", features = ["rt"] }
 rand = "0.9"
-- 
2.51.0

